book/version_control.md

```diff
@@ -403,10 +400,9 @@ Analytical projects might use issues to plan and discuss the steps involved in d
 
 [Setting issue templates](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository) for your project can be an effective way of encouraging users and collaborators to use informative descriptions when creating issues. For example, a bug issue should include simple instructions to help maintainers reproduce the problem. While feature requests might include information on how the user expects the new feature to work and details what problem it will help them to overcome.
 
-
-
 (pull-requests)=
-### Making the most of Pull Requests
+
+### Make the most of Pull Requests
```

Comment:
We should be specific about when or why, something like "Use Pull requests for reviewing changes"